packages/components/text/src/text.styles.ts

```diff
@@ -73,7 +107,9 @@ const getElementFontSize = (elementType?: string) => {
 export const bodyStyles = (props: TBodyProps) => css`
   ${getBaseStyles()}
   margin: 0;
-  font-size: 1rem;
+  font-size: ${designTokens.fontSizeForTextAsBody};
+  line-height: ${designTokens.lineHeightForTextAsBody};
+  font-weight: ${designTokens.fontWeightForTextAsBody};
   ${props.isBold && bold}
```

Comment:
Should we use a token also for `bold`?

Also, at moment bold is equivalent to `700`. Is it going to be the same for the new design? If so, it's missing in the token choices.